src/webui/api/appcontroller.cpp

```diff
@@ -83,12 +84,23 @@ void AppController::preferencesAction()
     QVariantMap data;
 
     // Downloads
-    // Hard Disk
+    // When adding a torrent
+    data["create_subfolder_enabled"] = session->isCreateTorrentSubfolder();
+    data["start_paused_enabled"] = session->isAddTorrentPaused();
+    data["auto_delete_mode"] = static_cast<int>(TorrentFileGuard::autoDeleteMode());
+    data["preallocate_all"] = session->isPreallocationEnabled();
+    data["incomplete_files_ext"] = session->isAppendExtensionEnabled();
+    // Saving Management
+    data["auto_tmm_enabled"] = session->isAutoTMMDisabledByDefault() ? 0 : 1;
```

Comment:
Why not `data["auto_tmm_enabled"] = session->isAutoTMMDisabledByDefault();`? Why do you convert it into number?
(below too)